drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -233,8 +236,18 @@ def studio_view(self, context):
         for js_url in js_urls:
             fragment.add_javascript_url(self.runtime.local_resource_url(self, js_url))
 
+        # Do a bit of manipulation so we get the appearance of a list of zone options on
+        # items that still have just a single zone stored
+        data = copy.deepcopy(self.data)
```

Comment:
@haikuginger Can we do without copying `self.data` here? My understanding is that we don't necessarily want to preserve `item["zone"]` entries for `data` belonging to old instances; we just want these instances to keep working :) So it would be OK for this code to replace `item["zone"]` with `item["zones"]` I think (moving us closer to being able to remove backward compatibility hacks over time).